docs/my-website/docs/providers/huggingface.md

```diff
@@ -2,466 +2,363 @@ import Image from '@theme/IdealImage';
-os.environ["HUGGINGFACE_API_KEY"] = "huggingface_api_key"
-
-messages = [{ "content": "I like you, I love you!","role": "user"}]
+os.environ["HF_TOKEN"] = "hf_xxxxxx"
 
-# e.g. Call 'shahrukhx01/question-vs-statement-classifier' hosted on HF Inference endpoints
 response = completion(
-  model="huggingface/text-classification/shahrukhx01/question-vs-statement-classifier",
-  messages=messages,
-  api_base="https://my-endpoint.endpoints.huggingface.cloud",
+    model="huggingface/together/deepseek-ai/DeepSeek-R1",
+    messages=[{"content": "How many r's are in the word `strawberry`?", "role": "user"}]
 )
-
 print(response)
 ```
 
-</TabItem> 
-<TabItem value="proxy" label="PROXY">
+### Streaming
+Now, let's see what a streaming request looks like.
 
-1. Add models to your config.yaml
+```python
+import os
+from litellm import completion
 
-```yaml
-model_list:
-  - model_name: bert-classifier
-    litellm_params:
-      model: huggingface/text-classification/shahrukhx01/question-vs-statement-classifier
-      api_key: os.environ/HUGGINGFACE_API_KEY
-      api_base: "https://my-endpoint.endpoints.huggingface.cloud"
-```
+os.environ["HF_TOKEN"] = "hf_xxxxxx"
 
-2. Start the proxy
+response = completion(
+    model="huggingface/together/deepseek-ai/DeepSeek-R1",
+    messages=[{"content": "How many r's are in the word `strawberry`?", "role": "user"}],
+    stream=True
+)
 
-```bash
-$ litellm --config /path/to/config.yaml --debug
+for chunk in response:
+    print(chunk)
 ```
 
-3. Test it!
+### Image Input
+You can also pass images to the model.
 
-```shell
-curl --location 'http://0.0.0.0:4000/chat/completions' \
-    --header 'Authorization: Bearer sk-1234' \
-    --header 'Content-Type: application/json' \
-    --data '{
-    "model": "bert-classifier",
-    "messages": [
-      {
-          "role": "user",
-          "content": "I like you!"
-      }
-      ],
-}'
-```
+```python
+from litellm import completion
 
-</TabItem> 
-</Tabs>
-</TabItem>
-<TabItem value="dedicated" label="Dedicated Inference Endpoints">
+# Set your Hugging Face Token
+os.environ["HF_TOKEN"] = "hf_xxxxxx"
+
+messages=[
+        {
+            "role": "user",
+            "content": [
+                {"type": "text", "text": "What's in this image?"},
+                {
+                    "type": "image_url",
+                    "image_url": {
+                        "url": "https://upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Gfp-wisconsin-madison-the-nature-boardwalk.jpg/2560px-Gfp-wisconsin-madison-the-nature-boardwalk.jpg",
+                    }
+                },
+            ],
+        }
+    ]
 
-Steps to use
-* Create your own Hugging Face dedicated endpoint here: https://ui.endpoints.huggingface.co/
-* Set `api_base` to your deployed api base
-* Add the `huggingface/` prefix to your model so litellm knows it's a huggingface Deployed Inference Endpoint
+response = completion(
+    model="huggingface/sambanova/Qwen/Qwen2.5-72B-Instruct", 
```

Comment:
```suggestion
    model="huggingface/sambanova/meta-llama/Llama-3.2-11B-Vision-Instruct", 
```

Let's use a VLM like https://huggingface.co/meta-llama/Llama-3.2-11B-Vision-Instruct  (https://huggingface.co/Qwen/Qwen2.5-72B-Instruct is not AFAIK)